testplan/web_ui/testing/src/Common/utils.js

```diff
@@ -96,10 +96,49 @@ function domToString(dom) {
  * @param {string} uid - string to be encoded
  * @returns {string}
  */
-function encodeURIComponent2 (str) {
+function encodeURIComponent2(str) {
   return encodeURIComponent(encodeURIComponent(str));
 }
 
+/**
```

Comment:
I wonder if there's any library we could use instead of implementing this / 60 & % 60 logic ourselves?

Reply:
If remember well that is just used in the api communication in interactive mode, I think we can switch it to base64url.